sdk/core/azure-core-rest/src/test/java/com/azure/android/core/rest/util/paging/CFBackedPageAsyncStream.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.core.rest.util.paging;
+
+import com.azure.android.core.logging.ClientLogger;
+import com.azure.android.core.util.Function;
+import com.azure.android.core.util.Predicate;
+import com.azure.android.core.util.AsyncStream;
+import com.azure.android.core.util.AsyncStreamHandler;
+import com.azure.android.core.util.CancellationToken;
+
+import java.util.concurrent.CancellationException;
+
+import java9.util.concurrent.CompletableFuture;
+import java9.util.concurrent.CompletionException;
+
+final class CFBackedPageAsyncStream<T>
```

Comment:
This is Java. What does "CF" stand for? :-)

Reply:
:) true. its stands for "CompletableFuture", this is a private class for testing, but the name can be more specific if we prefer so.